MonoGame.Framework.Content.Pipeline/Graphics/FontDescriptionStyle.cs

```diff
@@ -14,6 +14,11 @@ namespace Microsoft.Xna.Framework.Content.Pipeline.Graphics
     public enum FontDescriptionStyle
     {
         /// <summary>
+        /// Normal text.
+        /// </summary>
+        Regular=0,
```

Comment:
Test added in the latest commit, i'll need someone to run it in XNA as i don't have the XNA tooling installed.
Also, I have seen that SharpFont includes a similar enum, but with switched values for bold and italic